src/commands/deploy/document-store.ts

```diff
@@ -33,8 +33,10 @@ export const handler = async (args: DeployDocumentStoreCommand): Promise<string
     );
     return documentStore.contractAddress;
   } catch (e) {
-    if (e instanceof TypeError) {
+    if (e instanceof Error) {
```

Comment:
can we abstract into a getErrorMessage util (and put it into utils)?

https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript

something like that.